activities/views/follows.py

```diff
@@ -15,19 +17,25 @@ class FollowsPage(TemplateView):
 
     def get_context_data(self):
         # Gather all identities with a following relationship with us
-        identities = {}
-        for outbound_follow in self.request.identity.outbound_follows.filter(
-            state__in=FollowStates.group_active()
-        ):
-            identities.setdefault(outbound_follow.target, {})[
-                "outbound"
-            ] = outbound_follow
-        for inbound_follow in self.request.identity.inbound_follows.filter(
-            state__in=FollowStates.group_active()
-        ):
-            identities.setdefault(inbound_follow.source, {})["inbound"] = inbound_follow
+        follows = Follow.objects.filter(
```

Comment:
Could you move this to a ListView instead? See the "federation" admin page for a good example.

Reply:
Done at https://github.com/jointakahe/takahe/pull/170/commits/265ff7c50f3c261ef858fa61c46a0466dea5d49c